### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by kianmeng**
Found via `typos --format brief`

### CODE REVIEW

File: src/network/core_utils.rs [hunk 1]
```diff
@@ -388,7 +388,7 @@ pub fn add_default_routes(
 }
 
 pub fn disable_ipv6_autoconf(if_name: &str) -> NetavarkResult<()> {
-    // make sure autoconf is off, we want manaully config only
+    // make sure autoconf is off, we want manually config only
```
Comment (edsantiago) [inline src/network/core_utils.rs:391]: This should actually be `manual`, not manually.

**Review by edsantiago — CHANGES_REQUESTED** [review]
Thank you @kianmeng !

Changes LGTM with one exception. And, you need to [sign your commits](https://github.com/containers/podman/blob/main/CONTRIBUTING.md#sign-your-prs).

**Review by edsantiago — APPROVED** [review]
/hold cancel

Thank you!

**Comment by kianmeng** [comment]
:partying_face: :partying_face: :partying_face: :partying_face: :partying_face:

**Pull request merged**